src/index.js

```diff
@@ -110,4 +126,7 @@ window.addEventListener("beforeinstallprompt", e => {
 
   hideHero = false;
   remountUI();
+
+  await fetchFeaturedRooms();
+  setInterval(fetchFeaturedRooms, 30000);
```

Comment:
oh, this is a problem for hubs cloud now that I think about it. we gotta be really careful here, because if we have anything like this where a page, when left open, continually accesses the db, it means a single open page can keep their aurora instance hot. we now have an idle timeout in hubs and the admin console for this reason.

instead, we should either skip the polling, or stop polling after some reasonable interval like 15 minutes. i think just skipping it is probably the right tradeoff. note that the 'right' solution here would be to open up a new channel to fetch presence data that has no db access in its read path, so the database could shut down *and* we could get real time updates.

Reply:
Just removed the polling for now.